components/cards/ActionForm.js

```diff
@@ -353,14 +368,23 @@ export function ActionForm({
                 }
               />
             </Form.Group>
-            <Button
-              variant="primary"
-              className="w-100"
-              onClick={() => submit()}
-              disabled={isActing || isLoading || isInvalid}
-            >
-              {isActing || isLoading ? spinner : "Confirm"}
-            </Button>
+            <div style={{ display: "flex", gap: "16px" }}>
```

Comment:
Can you add some spacing between the top of the buttons and the text field?
<img width="359" alt="Screen Shot 2022-05-01 at 12 18 37" src="https://user-images.githubusercontent.com/6885592/166161165-8a869362-d9ae-4141-8aa6-d1d7840998d7.png">